inst/EpiModel2.rmd

```diff
@@ -422,7 +483,7 @@ control <- control.net(type = "SI", nsteps = 100, depend = TRUE,
 should be updated to:
```

Comment:
suggested changes to lines 466-480: note that if you choose to use "re-simulate" with hyphen you should change here too

> to group (".g2" suffixes)
> ```{r echo = TRUE, eval = FALSE}
> # EpiModel 2.0 Workflow
> ## Update all parameters and initial conditions from ".m2" to ".g2" language.
> param <- param.net(inf.prob = 0.4, inf.prob.g2 = 0.2)
> init <- init.net(i.num = 20, i.num.g2 = 20)
> ```
> Finally, control parameters `depend` and `save.network` (along with `save.transmat`) have been replaced. The control parameter `resimulate.network` has replaced `depend` when a user wishes to resimulate the network at each timestep of simulation. This change makes it clearer what the parameter is actually doing. The parameters `save.network` and `save.transmat` have been replaced with `tergmLite`: if a user wishes to save the network and transmission matrix, they must use the full estimation routine provided by `tergm` (`tergmLite == FALSE`); otherwise, `tergmLite` is used, and no network of transmission matrix is saved. Thus:
> ```{r echo = TRUE, eval = FALSE}
> # EpiModel 1.x 
> control <- control.net(type = "SI", nsteps = 100, depend = TRUE, 
>                        save.network = TRUE)
> ```